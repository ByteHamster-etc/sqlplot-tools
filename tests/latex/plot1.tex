line1
% IMPORT-DATA test test.data
line2
% PLOT SELECT LOG(testsize) / LOG(2) AS x, bandwidth AS y FROM test WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
line3
%% PLOT SELECT LOG(testsize) / LOG(2) AS x, bandwidth AS y FROM test
%% WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
\addplot[red] coordinates { (24,0.144) };
line4
{
indentation start
    %% PLOT SELECT testsize AS x, rate * 1e12 AS y FROM test
    %% WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
    text
    %% PLOT SELECT testsize AS x, rate * 1e12 AS y FROM test
    %% WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
    \addplot[red] coordinates { (24,0.144) };
indentation end
}
line5