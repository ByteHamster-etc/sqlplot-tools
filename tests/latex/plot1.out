line1
% IMPORT-DATA test test.data
line2
% PLOT SELECT testsize AS x, bandwidth AS y FROM test WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
\addplot coordinates { (1152,2.00049e+10) (2176,2.00592e+10) (3200,2.01016e+10) (4224,2.00787e+10) (6272,2.01033e+10) (8320,1.98443e+10) (12416,1.99341e+10) (16512,1.99704e+10) (20608,1.99977e+10) (24704,2.00171e+10) (28800,2.00244e+10) (32896,1.95887e+10) (41088,1.33525e+10) (49280,1.33527e+10) (65664,1.33532e+10) (98432,1.33536e+10) (131200,1.33535e+10) (196736,1.33537e+10) (262272,1.33538e+10) (393344,1.33544e+10) (524416,1.33554e+10) (786560,1.33571e+10) (1048704,1.33223e+10) (1310848,1.25999e+10) (1572992,1.25974e+10) (1835136,1.25935e+10) (2097280,1.18143e+10) (2359424,1.17322e+10) (2621568,1.11514e+10) (2883712,9.63556e+09) (3145856,8.83049e+09) (4194432,5.70303e+09) (5243008,3.76486e+09) (6291584,2.97082e+09) (7340160,2.62547e+09) (8388736,2.44873e+09) (9437312,2.35262e+09) (10485888,2.28865e+09) (12583040,2.22295e+09) (14680192,2.19541e+09) (16777344,2.1806e+09) (20971648,2.1704e+09) (25165952,2.16835e+09) (29360256,2.16741e+09) (33554560,2.16724e+09) (67108992,2.16707e+09) (1.34218e+08,2.1674e+09) (2.68436e+08,2.16717e+09) (5.36871e+08,2.16721e+09) (1.07374e+09,2.16729e+09) (2.14748e+09,2.16717e+09) (4.29497e+09,2.16733e+09) (8.58993e+09,2.16723e+09) (1.71799e+10,2.1669e+09) };
line3
%% PLOT SELECT testsize AS x, bandwidth AS y FROM test
%% WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
\addplot[red] coordinates { (1152,2.12566e+10) (2176,2.12068e+10) (3200,2.12568e+10) (4224,2.12565e+10) (6272,2.12549e+10) (8320,2.10335e+10) (12416,2.11052e+10) (16512,2.11423e+10) (20608,2.11649e+10) (24704,2.1178e+10) (28800,2.11881e+10) (32896,2.01671e+10) (41088,1.54346e+10) (49280,1.54457e+10) (65664,1.54706e+10) (98432,1.54763e+10) (131200,1.54782e+10) (196736,1.54527e+10) (262272,1.54039e+10) (393344,1.54242e+10) (524416,1.5424e+10) (786560,1.54265e+10) (1048704,1.54166e+10) (1310848,1.52702e+10) (1572992,1.52696e+10) (1835136,1.52691e+10) (2097280,1.43805e+10) (2359424,1.43101e+10) (2621568,1.36672e+10) (2883712,1.18576e+10) (3145856,1.09755e+10) (4194432,7.46265e+09) (5243008,5.23999e+09) (6291584,4.38139e+09) (7340160,4.03316e+09) (8388736,3.84756e+09) (9437312,3.74337e+09) (10485888,3.66671e+09) (12583040,3.58156e+09) (14680192,3.54338e+09) (16777344,3.52208e+09) (20971648,3.50742e+09) (25165952,3.50313e+09) (29360256,3.50257e+09) (33554560,3.50191e+09) (67108992,3.50248e+09) (1.34218e+08,3.50222e+09) (2.68436e+08,3.50208e+09) (5.36871e+08,3.50212e+09) (1.07374e+09,3.50154e+09) (2.14748e+09,3.50165e+09) (4.29497e+09,3.50154e+09) (8.58993e+09,3.50111e+09) (1.71799e+10,3.50009e+09) };
line4
{
indentation start
    %% PLOT SELECT testsize AS x, rate * 1e12 AS y FROM test
    %% WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
    \addplot coordinates { (1152,399.903) (2176,398.82) (3200,397.977) (4224,398.431) (6272,397.944) (8320,403.139) (12416,401.323) (16512,400.594) (20608,400.046) (24704,399.658) (28800,399.513) (32896,408.398) (41088,599.137) (49280,599.132) (65664,599.107) (98432,599.089) (131200,599.094) (196736,599.085) (262272,599.081) (393344,599.055) (524416,599.011) (786560,598.932) (1048704,600.497) (1310848,634.925) (1572992,635.052) (1835136,635.248) (2097280,677.147) (2359424,681.886) (2621568,717.397) (2883712,830.258) (3145856,905.952) (4194432,1402.76) (5243008,2124.91) (6291584,2692.86) (7340160,3047.08) (8388736,3267) (9437312,3400.47) (10485888,3495.51) (12583040,3598.82) (14680192,3643.96) (16777344,3668.71) (20971648,3685.95) (25165952,3689.44) (29360256,3691.05) (33554560,3691.34) (67108992,3691.62) (1.34218e+08,3691.06) (2.68436e+08,3691.45) (5.36871e+08,3691.39) (1.07374e+09,3691.25) (2.14748e+09,3691.45) (4.29497e+09,3691.17) (8.58993e+09,3691.34) (1.71799e+10,3691.91) };
    text
    %% PLOT SELECT testsize AS x, rate * 1e12 AS y FROM test
    %% WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
    \addplot[red] coordinates { (1152,376.353) (2176,377.238) (3200,376.351) (4224,376.356) (6272,376.383) (8320,380.345) (12416,379.053) (16512,378.388) (20608,377.984) (24704,377.751) (28800,377.571) (32896,396.686) (41088,518.316) (49280,517.942) (65664,517.11) (98432,516.92) (131200,516.856) (196736,517.708) (262272,519.348) (393344,518.666) (524416,518.673) (786560,518.587) (1048704,518.921) (1310848,523.896) (1572992,523.916) (1835136,523.934) (2097280,556.31) (2359424,559.047) (2621568,585.343) (2883712,674.674) (3145856,728.899) (4194432,1072) (5243008,1526.72) (6291584,1825.91) (7340160,1983.55) (8388736,2079.24) (9437312,2137.11) (10485888,2181.79) (12583040,2233.66) (14680192,2257.73) (16777344,2271.38) (20971648,2280.88) (25165952,2283.67) (29360256,2284.04) (33554560,2284.47) (67108992,2284.1) (1.34218e+08,2284.27) (2.68436e+08,2284.36) (5.36871e+08,2284.33) (1.07374e+09,2284.71) (2.14748e+09,2284.64) (4.29497e+09,2284.71) (8.58993e+09,2284.99) (1.71799e+10,2285.66) };
indentation end
}
line5
